src/core/positioning/internalgnssreceiver.cpp

```diff
@@ -125,7 +125,7 @@ void InternalGnssReceiver::handlePositionUpdated( const QGeoPositionInfo &positi
                                                             speed,
                                                             direction,
                                                             mSatellitesInfo, 0, 0, 0,
-                                                            hacc,
+                                                            0.01, //hacc,
```

Comment:
What's that?

Reply:
That sir is the only way to test things out on a machine that doesnt have a gnss device ;) I'll revert.